src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js

```diff
@@ -228,20 +228,16 @@ export default class ListStore {
             return {};
         }
 
-        const schemaSettings = ListStore.getSchemaSetting(this.listKey, this.userSettingsKey);
-        const schema = this.schema;
-        if (!schemaSettings) {
-            return schema;
-        }
+        const schemaSettings = ListStore.getSchemaSetting(this.listKey, this.userSettingsKey) || [];
 
-        const userSchema = {};
+        const userSchema = {...this.schema};
```

Comment:
This completely overrides any ordering done by the user, doesn't it? I think we should start with the `schemaSettings` variable from above, and only add what is missing at the end, shouldn't we?

And I just tested, with this change it is not possible to sort the columns at all.

Reply:
did not think about the order at all 🙈 will adjust it